lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -6782,7 +6789,6 @@ class PullbackEmitter final : public SILInstructionVisitor<PullbackEmitter> {
       src = cvi->getOperand();
     addAdjointValue(si->getParent(), src,
                     makeConcreteAdjointValue(newAdjValue), si->getLoc());
-    blockTemporaries[ai->getParent()].push_back(newAdjValue);
```

Comment:
Note: this line is redundant because `recordTemporary(newAdjValue)` is called a few lines above.
`blockTemporaries[ai->getParent()]` is also incorrect because `blockTemporaries` should be indexed by pullback bb, not original bb.